cloudevents/sdk/event/base.py

```diff
@@ -20,61 +20,141 @@
 # TODO(slinkydeveloper) is this really needed?
 class EventGetterSetter(object):
 
+    # ce-specversion
     def CloudEventVersion(self) -> str:
         raise Exception("not implemented")
 
-    # CloudEvent attribute getters
-    def EventType(self) -> str:
-        raise Exception("not implemented")
+    @property
+    def specversion(self):
+        return self.CloudEventVersion()
 
-    def Source(self) -> str:
+    def SetCloudEventVersion(self, specversion: str) -> object:
         raise Exception("not implemented")
 
-    def EventID(self) -> str:
-        raise Exception("not implemented")
+    @specversion.setter
+    def specversion(self, value: str):
+        self.SetCloudEventVersion(value)
```

Comment:
I believe having 3 methods like this is too much, don't we just need 2, getter and setter/`@property`/`@foo.setter`?

Reply:
The reason why there's 4 methods (property/foo.setter and getter/setter) is the original developers made getters/setters. To make sure I didn't break any existing code, I ade to make sure the old getters/setters still work which is why I didn't remove them. The benefit of the properties is now you can do something such as event.specversion which is more pythonic instead of event.CloudEventVersion()